mitiq/ddd/tests/test_ddd.py

```diff
@@ -226,3 +231,14 @@ def exec_xx_small_spacing(circuit):
     # What is important to test is getting different results.
     assert not np.isclose(unmitigated, mitigated_small_spacing)
     assert not np.isclose(mitigated_large_spacing, mitigated_small_spacing)
+
+
+@mark.parametrize("num_trials", [1, 3, 5])
```

Comment:
Since this test should be fast (correct me if i'm wrong) can we add a few more trials? Say up to 30?